frontend/src/components/UserList/UserList.tsx

```diff
@@ -64,12 +64,16 @@ const mapStateToProps = ({
   userData,
   usersData,
 }: StoreState): { usersWithFollows: User[] } => {
+  const logged_in_user = userData.data;
   const users = usersData.data || [];
 
   let following = userData.data?.following || [];
   let following_ids: number[] = following?.map((f) => f['to_id']);
 
-  const usersWithFollows = users.map((user) => {
+  const usersWithFollows = users.flatMap((user) => {
+    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap#for_adding_and_removing_items_during_a_map
```

Comment:
Swap places of the comment and function
```suggestion
    // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/flatMap#for_adding_and_removing_items_during_a_map
  const usersWithFollows = users.flatMap((user) => {
```